src/index.html

```diff
@@ -18,9 +18,9 @@
     <div id="waiting-container">
         <div class="waitingContent">
             <img class="logo" src="img/logo.svg" />
-            <h1>Ready to cast</h1>
-            <h2>Select your media in Jellyfin and play it here</h2>
+            Ready to cast
```

Comment:
Why remove ``h1``?

Reply:
It forces newline, alternately I could have tried the image within a h1 tag, but ultimately the css styling provides everthing we need